modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -3,85 +3,85 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<filter1>>
-//* <<filter2>>
-
 
 [[filter1]]
 == filter(Array<T>, (item: T, index: Number) -> Boolean): Array<T>
 
-Returns an array that containing elements that meet the criteria specified
-by a function (a lambda). The function is invoked with two parameters: `value`
-and `index`.
+Filters a list (array) leaving the items that meet the criteria specified by an expression.
+
+
+The expression must return `true` or `false`.
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name | Description
+| `items` | The list (array) to filter.
```

Comment:
Seriously, we don't need list & array, and only one of them can be technically true.